.gitignore

```diff
@@ -1 +1,2 @@
 target/
+.DS_Store
```

Comment:
This should be in your user's gitignore (~/.config/git/ignore) since it's specific to your own environment and not to the project.
See https://git-scm.com/docs/gitignore